admin/worker/run_autoscaler.go

```diff
@@ -48,20 +49,30 @@ func (w *Worker) runAutoscaler(ctx context.Context) error {
 				zap.Float64("scale_threshold_percentage", scaleThreshold),
 				zap.String("project_id", targetProject.ID),
 			)
-			break
+			continue
 		}
 
-		opts := &database.UpdateProjectOptions{
-			ProdSlots: rec.RecommendedSlots,
-		}
+		updatedProject, err := w.admin.UpdateProject(ctx, targetProject, &database.UpdateProjectOptions{
+			ProdSlots:            rec.RecommendedSlots,
+			Name:                 targetProject.Name,
+			Description:          targetProject.Description,
+			Public:               targetProject.Public,
+			GithubURL:            targetProject.GithubURL,
+			GithubInstallationID: targetProject.GithubInstallationID,
+			ProdVersion:          targetProject.ProdVersion,
+			ProdBranch:           targetProject.ProdBranch,
+			ProdVariables:        targetProject.ProdVariables,
+			ProdDeploymentID:     targetProject.ProdDeploymentID,
+			ProdTTLSeconds:       targetProject.ProdTTLSeconds,
+			Provisioner:          targetProject.Provisioner,
+			Annotations:          targetProject.Annotations,
+		})
 
-		updatedProject, err := w.admin.UpdateProject(ctx, targetProject, opts)
 		if err != nil {
 			w.logger.Error("failed to autoscale:", zap.String("project_id", rec.ProjectID), zap.Error(err))
-			break
+		} else {
+			w.logger.Info("succeeded in autoscaling:", zap.String("project_id", updatedProject.Name), zap.Int("project_slots", updatedProject.ProdSlots))
```

Comment:
Switching from `Debug` to `Info` here as I think it better leave logs for autoscaling successful events